src/app/clusters/bindings/PendingNotificationMap.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <assert.h>
+
+#include <app/clusters/bindings/PendingNotificationMap.h>
+#include <app/util/binding-table.h>
+
+namespace chip {
+
+int16_t PendingNotificationMap::FindLRUBindingEntryIndex()
+{
+    uint8_t bindingWithSamePeer[EMBER_BINDING_TABLE_SIZE];
```

Comment:
How big will this stack use be?

Reply:
The binding table size will be typicall 10 to 20.